helm/polaris/values.yaml

```diff
@@ -0,0 +1,370 @@
+#  - key: "node-role.kubernetes.io/control-plane"
+#    operator: "Exists"
+#    effect: "NoSchedule"
+
+# -- Affinity and anti-affinity for polaris pods. See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity.
+affinity: {}
+#  podAffinity:
+#    preferredDuringSchedulingIgnoredDuringExecution:
+#      - weight: 100
+#        podAffinityTerm:
+#          topologyKey: kubernetes.io/hostname
+#          labelSelector:
+#            matchExpressions:
+#              - key: app.kubernetes.io/name
+#                operator: In
+#                values:
+#                  - polaris
+
+# -- Configures the liveness probe for polaris pods.
+livenessProbe:
+  # -- Number of seconds after the container has started before liveness probes are initiated. Minimum value is 0.
+  initialDelaySeconds: 5
+  # -- How often (in seconds) to perform the probe. Minimum value is 1.
+  periodSeconds: 10
+  # -- Minimum consecutive successes for the probe to be considered successful after having failed. Minimum value is 1.
+  successThreshold: 1
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded. Minimum value is 1.
+  failureThreshold: 3
+  # -- Number of seconds after which the probe times out. Minimum value is 1.
+  timeoutSeconds: 10
+  # -- Optional duration in seconds the pod needs to terminate gracefully upon probe failure. Minimum value is 1.
+  terminationGracePeriodSeconds: 30
+
+# -- Configures the readiness probe for polaris pods.
+readinessProbe:
+  # -- Number of seconds after the container has started before readiness probes are initiated. Minimum value is 0.
+  initialDelaySeconds: 5
+  # -- How often (in seconds) to perform the probe. Minimum value is 1.
+  periodSeconds: 10
+  # -- Minimum consecutive successes for the probe to be considered successful after having failed. Minimum value is 1.
+  successThreshold: 1
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded. Minimum value is 1.
+  failureThreshold: 3
+  # -- Number of seconds after which the probe times out. Minimum value is 1.
+  timeoutSeconds: 10
+
+# -- Advanced configuration via Environment Variables.
+# Extra environment variables to add to the Polaris server container.
+# You can pass here any valid EnvVar object:
+# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#envvar-v1-core
+# This can be useful to get configuration values from Kubernetes secrets or config maps.
+extraEnv:
+  []
+#  - name: AWS_STORAGE_BUCKET
+#    value: s3://xxxxx/
+#  - name: AWS_ACCESS_KEY_ID
+#    valueFrom:
+#      secretKeyRef:
+#        name: aws-secret
+#        key: access_key_id
+#  - name: AWS_SECRET_ACCESS_KEY
+#    valueFrom:
+#      secretKeyRef:
+#        name: aws-secret
+#        key: secret_access_key
+
+# -- Configures for persistence.xml (ensure the key name is 'persistence.xml')
+persistenceConfigSecret: polaris-secret
```

Comment:
We should probably require the user specify a value for this secret since it's not present by default.

Reply:
Yes, this one is identically left out and not part of helm chart as it will contains the db auth/connection info for the backend. This is left as default and users can change it to whatever they like. It is not hard-coded on the actual ref on the other end. This is similar to how nessie is handling it:
```
# Cassandra settings. Only required when using CASSANDRA version store type; ignored otherwise.
cassandra:
  keyspace: nessie
  # -- The contact points for the Cassandra cluster. At least one contact point must be provided,
  # but more can be added for redundancy. The format is a comma-separated list of host:port elements.
  contactPoints: cassandra.cassandra.svc.cluster.local:9042
  localDatacenter: datacenter1
  secret:
    # -- The secret name to pull Cassandra credentials from.
    name: cassandra-creds
    # -- The secret key storing the Cassandra username.
    username: cassandra_username
    # -- The secret key storing the Cassandra password.
    password: cassandra_password
```
But if we have a better value to use, I can change it.